_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  "Project Tardigrade Is Ready For Early Users/Contributors"
+author: Andrii Rosa, Lukasz Osipiuk, Zebing Lin, Brian Zhan
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. 
+It’s available right now! Read on to learn how Project Tardigrade works. 
+It required some significant and fascinating engineering to get us there. 
+I think you’ll enjoy hearing about it.
+
+ <!--more-->
+
+## What is Project Tardigrade
+
+If you’ve done big data analysis before, you probably had to guess which engine to to use based on the expected memory load. 
```

Comment:
It is a very indirect way of saying that Trino can  not support queries which require a lot of memory while "other" engines can. 
IMO saying it explicitly - maybe using a bit softer wording than "can" and "can not" - would make this paragraph easier to consume.

Reply:
Good point. Rewrote the `What is Project Tardigrade` secion.